tests/unit/Actions.spec.ts

```diff
@@ -94,10 +94,10 @@ describe('Actions component', () => {
     store.state.stateModel.createShareStructureStep = { valid: true }
     store.state.stateModel.incorporationAgreementStep = { valid: true }
     store.state.stateModel.incorporationDateTime = { valid: true }
-    await Vue.nextTick(() => {
-      // verify File and Pay button state
-      expect(wrapper.find('#file-pay-btn').attributes('disabled')).toBeUndefined()
-    })
+    await Vue.nextTick()
+
+    // verify File and Pay button state
+    expect(wrapper.find('#file-pay-btn').attributes('disabled')).toBeUndefined()
```

Comment:
Good change, i found sometimes when wrapping the test in a nextTick callback it would artificially pass the test but throw the errors to the terminal.

Reply:
Right. The way around that is to specify `done` as a callback method for the test (instead of `()`), and then you have to call `done()` at the end of the arrow function. But if the test fails you just get "test timed out" instead of the actual expect error, so async is way better.